src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -385,7 +385,9 @@ public function storeDataArchive($fileName, array $tags)
         $uploadOptions->setIsPermanent(false);
         $uploadOptions->setIsPublic(false);
         $uploadOptions->setNotify(false);
-        $this->clientWrapper->getBasicClient()->uploadFile($zipFileName, $uploadOptions);
+
+        $this->clientWrapper->getTableAndFileStorageClient()->uploadFile($zipFileName, $uploadOptions);
```

Comment:
Vevnitř se to automaticky rozhoduje podle client options. Ty se v Runneru sestavujou z jobu https://github.com/keboola/job-runner/blob/main/src/Helper/BuildBranchClientOptionsHelper.php#L16

Takže pro sox by to mělo nově používat branchovanýho clienta. Pro klasický projekt basic.